pkg/kubectl/cmd/util/helpers.go

```diff
@@ -225,6 +225,17 @@ func GetFlagString(cmd *cobra.Command, flag string) string {
 	return s
 }
 
+func CheckDeprecatedGlobalFlags(cmd *cobra.Command) {
```

Comment:
Thanks. It's now removed and I use `MarkDeprecated` in overrides.go (when binding flags) instead.